win standalone examples

wlemkows opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/pmem/nvml/1948)
<!-- Reviewable:end -->

plebioda commented:
Please resolve conflicts

wlemkows commented:
done

---

Review status: 0 of 79 files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/nvml/1948#-:-Knn0y2U4YJnFF0E9w6W:bb7f2mf)*
<!-- Sent from Reviewable.io -->

krzycz commented:
Reviewed 3 of 12 files at r1, 75 of 78 files at r2, 2 of 2 files at r3.
Review status: all files reviewed at latest revision, 23 unresolved discussions.

---

*[src/examples/libpmem/full_copy.vcxproj, line 43 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPY_vpIRhD79IVmqlu:-KpPY_vpIRhD79IVmqlv:b-bgg2gj) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmem/full_copy.vcxproj#L43)):*
> ```XML
>   <ItemDefinitionGroup Condition="'$(SolutionName)'=='Examples'">
>     <Link>
>       <AdditionalDependencies>libpmem.lib</AdditionalDependencies>
> ```

is this required?
(same applies to all the other projects)

---

*[src/examples/libpmem/full_copy.vcxproj, line 48 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPZLsZ2EeJCvMPcxur:-KpPZLsZ2EeJCvMPcxus:b-enskr2) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmem/full_copy.vcxproj#L48)):*
> ```XML
>   <ItemDefinitionGroup>
>     <Manifest>
>       <AdditionalManifestFiles>..\..\LongPath.manifest</AdditionalManifestFiles>
> ```

If we zip the examples directory, the manifest file won't be included.
Shouldn't we have a copy in examples, or integrate it with example .props?
(same applies to all the other projects)

---

*[src/examples/libpmemblk/assetdb/asset_checkin.vcxproj, line 44 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpP_QLueMgZxAwee9kI:-KpP_QLueMgZxAwee9kJ:b-l6mwxj) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemblk/assetdb/asset_checkin.vcxproj#L44)):*
> ```XML
>   </PropertyGroup>
>   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
>   <ImportGroup Label="ExtensionSettings">
> ```

remove

---

*[src/examples/libpmemblk/assetdb/asset_list.vcxproj, line 64 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpP_ZvXsLSN_ia1D95C:-KpP_ZvXsLSN_ia1D95D:b-ndwyez) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemblk/assetdb/asset_list.vcxproj#L64)):*
> ```XML
>     <Import Project="..\..\Examples_$(Configuration).props" />
>   </ImportGroup>
>   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
> ```

These settings should be in .props. no need to duplicate them here.

---

*[src/examples/libpmemlog/logfile/printlog.c, line 87 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPe4zpE6iWX9R1ke3N:-KpPe4zpE6iWX9R1ke3O:bw86jmk) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemlog/logfile/printlog.c#L87)):*
> ```C
> 
> 	if (argc > 2) {
> 		if (strcmp(argv[1], "t") == 0) {
> ```

why not "-t"?

---

*[src/examples/libpmemlog/logfile/printlog.vcxproj, line 49 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPWhCv35XEXvp3qBa6:-KpPWhCv35XEXvp3qBa7:b-ppt0qd) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemlog/logfile/printlog.vcxproj#L49)):*
> ```XML
>   </ImportGroup>
>   <PropertyGroup>
>     <IncludePath>$(ProjectDir)..\..\;$(solutionDir)include;$(IncludePath);$(WindowsSDK_IncludePath);</IncludePath>
> ```

should it be here, or in .props?

---

*[src/examples/libpmemobj/doc_snippets/doc_snippets.vcxproj, line 15 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPdCKrL-TmI-CfUgUv:-KpPdCKrL-TmI-CfUgUw:bkazemv) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemobj/doc_snippets/doc_snippets.vcxproj#L15)):*
> ```XML
>   <PropertyGroup Label="Globals">
>     <ProjectGuid>{8DB4158E-96EE-4DE5-A425-C9638F50B9ED}</ProjectGuid>
>     <RootNamespace>pmemobj_cpp</RootNamespace>
> ```

why not just "pmemobj"?

---

*[src/examples/libpmemobj/map/libmap.vcxproj, line 74 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPaxUHqDEOeUk1yWX-:-KpPaxUHqDEOeUk1yWX0:buttl1r) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemobj/map/libmap.vcxproj#L74)):*
> ```XML
>   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.props" />
>   <ImportGroup Label="PropertySheets">
>     <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
> ```

is this required?

---

*[src/examples/libpmemobj/pmemlog/obj_pmemlog.vcxproj, line 66 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPbO1dI7wfkGb3vZ17:-KpPbO1dI7wfkGb3vZ18:b-kibm5w) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemobj/pmemlog/obj_pmemlog.vcxproj#L66)):*
> ```XML
>   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
>     <ClCompile>
>       <CompileAs>CompileAsC</CompileAs>
> ```

is this required here? should be in .props

---

*[src/examples/libpmemobj/pmemlog/obj_pmemlog_minimal.vcxproj, line 66 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPbYUAaMdw8OqbtuJK:-KpPbYUAaMdw8OqbtuJL:b-7ropln) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemobj/pmemlog/obj_pmemlog_minimal.vcxproj#L66)):*
> ```XML
>   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
>     <ClCompile>
>       <CompileAs>CompileAsC</CompileAs>
> ```

ditto

---

*[src/examples/libpmemobj/string_store_tx/reader.vcxproj, line 15 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPc24u0s9FllS_sGrR:-KpPc24u0s9FllS_sGrS:bkazemv) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemobj/string_store_tx/reader.vcxproj#L15)):*
> ```XML
>   <PropertyGroup Label="Globals">
>     <ProjectGuid>{59D7A9CD-9912-40E4-96E1-8A873F777F62}</ProjectGuid>
>     <RootNamespace>pmemobj_tx</RootNamespace>
> ```

why not just "pmemobj"?

---

*[src/examples/libpmemobj/string_store_tx/writer.vcxproj, line 15 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPc5fewwtmpROEUj8n:-KpPc5fewwtmpROEUj8o:b-7ropln) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemobj/string_store_tx/writer.vcxproj#L15)):*
> ```XML
>   <PropertyGroup Label="Globals">
>     <ProjectGuid>{7337E34A-97B0-44FC-988B-7E6AE7E0FBBF}</ProjectGuid>
>     <RootNamespace>pmemobj_tx</RootNamespace>
> ```

ditto

---

*[src/examples/libpmemobj/string_store_tx_type/writer.vcxproj, line 15 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPc8UnAcl7Lp6VLH5i:-KpPc8UnAcl7Lp6VLH5j:b-7ropln) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemobj/string_store_tx_type/writer.vcxproj#L15)):*
> ```XML
>   <PropertyGroup Label="Globals">
>     <ProjectGuid>{1EB3DE5B-6357-498D-8CAC-EEC0209EA454}</ProjectGuid>
>     <RootNamespace>pmemobj_tx_type</RootNamespace>
> ```

ditto

---

*[src/test/ex_libpmemblk/TEST0.PS1, line 35 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPTdDQU1X2eow540G_:-KpPTdDQU1X2eow540Ga:b2nh8oq) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/test/ex_libpmemblk/TEST0.PS1#L35)):*
> ```
> 
> #
> # src\test\ex_libpmemblk\TEST0 -- unit test for libpmemblk examples
> ```

???

---

*[src/test/ex_libpmemblk/TEST0.PS1, line 42 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPTdzU-HSAY4qqPzpX:-KpPTdzU-HSAY4qqPzpY:b2nh8oq) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/test/ex_libpmemblk/TEST0.PS1#L42)):*
> ```
>     $DIR = ""
>     )
> $Env:UNITTEST_NAME = "ex_libpmemblk\TEST0"
> ```

???

---

*[src/test/ex_libpmemlog/TEST0.PS1, line 42 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPTigZ2zp0FbiDItbH:-KpPTigZ2zp0FbiDItbI:b2nh8oq) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/test/ex_libpmemlog/TEST0.PS1#L42)):*
> ```
>     $DIR = ""
>     )
> $Env:UNITTEST_NAME = "ex_libpmemlog\TEST0"
> ```

???

---

*[src/test/ex_libpmemobj/TEST17.PS1, line 34 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPTGWkz6zIQAKeQhT_:-KpPTGWkz6zIQAKeQhTa:b2nh8oq) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/test/ex_libpmemobj/TEST17.PS1#L34)):*
> ```
> 
> #
> # src\test\ex_libpmemobj\TEST17 -- unit test for libpmemobj examples
> ```

???

---

*[src/test/ex_libpmemobj/TEST17.PS1, line 41 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPTHmEADc34kF5PWPu:-KpPTHmEADc34kF5PWPv:b2nh8oq) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/test/ex_libpmemobj/TEST17.PS1#L41)):*
> ```
>     $DIR = ""
>     )
> $Env:UNITTEST_NAME = "ex_libpmemobj\TEST17"
> ```

???

---

*[src/test/ex_libpmemobj/TEST19.PS1, line 41 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPTL1Hdp0FvA5iStV5:-KpPTL1Hdp0FvA5iStV6:b2nh8oq) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/test/ex_libpmemobj/TEST19.PS1#L41)):*
> ```
>     $DIR = ""
>     )
> $Env:UNITTEST_NAME = "ex_libpmemobj\TEST19"
> ```

???

---

*[src/test/ex_libpmemobj/TEST19.PS1, line 62 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPTPvHnPJJEd3VKpG3:-KpPTPvHnPJJEd3VKpG4:b-j0d31j) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/test/ex_libpmemobj/TEST19.PS1#L62)):*
> ```
> c 4321
> q
> ```

is this intentional?

---

*[utils/CREATE-ZIP.PS1, line 54 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPgHzndw1TSiL9I1wn:-KpPgHzndw1TSiL9I1wo:b42v3ox) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/utils/CREATE-ZIP.PS1#L54)):*
> ```
> $srcdir = $rootdir + "\src\"
> $zipdir = $builddir + "\nvml\"
> $zipexp = $rootdir + "\nvml_examples\"
> ```

$zipexpdir ?

---

*[utils/CREATE-ZIP.PS1, line 107 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPgVSmbgZXNFvcLnrc:-KpPgVSmbgZXNFvcLnrd:b-cnpz4v) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/utils/CREATE-ZIP.PS1#L107)):*
> ```
> $comprlevel = [System.IO.Compression.CompressionLevel]::Optimal
> 
> if($build -eq "release") {
> ```

"setup" ?

---

*[utils/CREATE-ZIP.PS1, line 135 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPg8hznFynnNG1Un5d:-KpPg8hznFynnNG1Un5e:basczvv) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/utils/CREATE-ZIP.PS1#L135)):*
> ```
> 
> Remove-Item $zipdir -Force -Recurse -ea si
> Remove-Item $zipdir -Force -Recurse -ea si
> ```

did you mean $zipexp ?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/nvml/1948)*
<!-- Sent from Reviewable.io -->

wlemkows commented:
Review status: 28 of 78 files reviewed at latest revision, 23 unresolved discussions.

---

*[src/examples/libpmem/full_copy.vcxproj, line 43 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPY_vpIRhD79IVmqlu:-KpUGk047Ni_k5kOI43q:bmgbxnx) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmem/full_copy.vcxproj#L43)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

is this required?
(same applies to all the other projects)
</blockquote></details>

It is not. removed.

---

*[src/examples/libpmem/full_copy.vcxproj, line 48 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPZLsZ2EeJCvMPcxur:-KpU3ICdR6Ny7xQqCllP:b-kmux37) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmem/full_copy.vcxproj#L48)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

If we zip the examples directory, the manifest file won't be included.
Shouldn't we have a copy in examples, or integrate it with example .props?
(same applies to all the other projects)
</blockquote></details>

We have a copy of manifest in zip.
Look at CREATE-ZIP.PS1 line 110

---

*[src/examples/libpmemblk/assetdb/asset_checkin.vcxproj, line 44 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpP_QLueMgZxAwee9kI:-KpUJSoFyqCsDxOcWDpr:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemblk/assetdb/asset_checkin.vcxproj#L44)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

remove
</blockquote></details>

Done.

---

*[src/examples/libpmemblk/assetdb/asset_list.vcxproj, line 64 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpP_ZvXsLSN_ia1D95C:-KpUOLLJ5HxG8HrdQ6Aw:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemblk/assetdb/asset_list.vcxproj#L64)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

These settings should be in .props. no need to duplicate them here.
</blockquote></details>

Done.

---

*[src/examples/libpmemlog/logfile/printlog.c, line 87 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPe4zpE6iWX9R1ke3N:-KpUTNZydA3PZfBBICZX:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemlog/logfile/printlog.c#L87)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

why not "-t"?
</blockquote></details>

Done.

---

*[src/examples/libpmemlog/logfile/printlog.vcxproj, line 49 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPWhCv35XEXvp3qBa6:-KpUkRO4rUKwoQtpr7K5:bb7hf91) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemlog/logfile/printlog.vcxproj#L49)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

should it be here, or in .props?
</blockquote></details>

here

---

*[src/examples/libpmemobj/doc_snippets/doc_snippets.vcxproj, line 15 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPdCKrL-TmI-CfUgUv:-KpUpi9YCQjKCffP0x7F:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemobj/doc_snippets/doc_snippets.vcxproj#L15)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

why not just "pmemobj"?
</blockquote></details>

Done.

---

*[src/examples/libpmemobj/map/libmap.vcxproj, line 74 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPaxUHqDEOeUk1yWX-:-KpUYraru7A_2DfxWDdK:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemobj/map/libmap.vcxproj#L74)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

is this required?
</blockquote></details>

Done.

---

*[src/examples/libpmemobj/pmemlog/obj_pmemlog.vcxproj, line 66 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPbO1dI7wfkGb3vZ17:-KpU_agymC9hbOhVbQUX:b6c5qk6) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemobj/pmemlog/obj_pmemlog.vcxproj#L66)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

is this required here? should be in .props
</blockquote></details>

It is not required here at all. removed.

---

*[src/examples/libpmemobj/pmemlog/obj_pmemlog_minimal.vcxproj, line 66 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPbYUAaMdw8OqbtuJK:-KpUlizd2k4uc3sOGh-X:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemobj/pmemlog/obj_pmemlog_minimal.vcxproj#L66)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

ditto
</blockquote></details>

Done.

---

*[src/examples/libpmemobj/string_store_tx/reader.vcxproj, line 15 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPc24u0s9FllS_sGrR:-KpUmyNFUQwQUtRHxJ8l:b-d0kb12) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemobj/string_store_tx/reader.vcxproj#L15)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

why not just "pmemobj"?
</blockquote></details>

To distinguish readers/writers output exe files.

---

*[src/examples/libpmemobj/string_store_tx/writer.vcxproj, line 15 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPc5fewwtmpROEUj8n:-KpUmzaWNXhmKmHLl2MQ:b-d0kb12) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemobj/string_store_tx/writer.vcxproj#L15)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

ditto
</blockquote></details>

To distinguish readers/writers output exe files.

---

*[src/examples/libpmemobj/string_store_tx_type/writer.vcxproj, line 15 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPc8UnAcl7Lp6VLH5i:-KpUn-S618MrolzA6j8Q:b-d0kb12) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/examples/libpmemobj/string_store_tx_type/writer.vcxproj#L15)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

ditto
</blockquote></details>

To distinguish readers/writers output exe files.

---

*[src/test/ex_libpmemblk/TEST0.PS1, line 35 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPTdDQU1X2eow540G_:-KpUaSwZMC2oY1bGzGIN:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/test/ex_libpmemblk/TEST0.PS1#L35)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

???
</blockquote></details>

Done.

---

*[src/test/ex_libpmemblk/TEST0.PS1, line 42 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPTdzU-HSAY4qqPzpX:-KpUaTJY-ilcYX6eELRT:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/test/ex_libpmemblk/TEST0.PS1#L42)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

???
</blockquote></details>

Done.

---

*[src/test/ex_libpmemlog/TEST0.PS1, line 42 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPTigZ2zp0FbiDItbH:-KpUagcjwLkafSjV2qOO:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/test/ex_libpmemlog/TEST0.PS1#L42)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

???
</blockquote></details>

Done.

---

*[src/test/ex_libpmemobj/TEST17.PS1, line 34 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPTGWkz6zIQAKeQhT_:-KpUatzSSuXyNTilIwm_:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/test/ex_libpmemobj/TEST17.PS1#L34)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

???
</blockquote></details>

Done.

---

*[src/test/ex_libpmemobj/TEST17.PS1, line 41 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPTHmEADc34kF5PWPu:-KpUauP5wNLGRzHMwMGX:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/test/ex_libpmemobj/TEST17.PS1#L41)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

???
</blockquote></details>

Done.

---

*[src/test/ex_libpmemobj/TEST19.PS1, line 41 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPTL1Hdp0FvA5iStV5:-KpUbIOFHRSqI1F75fNd:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/test/ex_libpmemobj/TEST19.PS1#L41)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

???
</blockquote></details>

Done.

---

*[src/test/ex_libpmemobj/TEST19.PS1, line 62 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPTPvHnPJJEd3VKpG3:-KpUbHs-J1c9hXDl-nY5:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/src/test/ex_libpmemobj/TEST19.PS1#L62)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

is this intentional?
</blockquote></details>

Done.

---

*[utils/CREATE-ZIP.PS1, line 54 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPgHzndw1TSiL9I1wn:-KpUeTJBLTlvXjLhe_m_:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/utils/CREATE-ZIP.PS1#L54)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

$zipexpdir ?
</blockquote></details>

Done.

---

*[utils/CREATE-ZIP.PS1, line 107 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPgVSmbgZXNFvcLnrc:-KpUenNOzAFiHc3xomEs:b83y3vu) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/utils/CREATE-ZIP.PS1#L107)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

"setup" ?
</blockquote></details>

It doesn't matter. It was easier to implement for release.

---

*[utils/CREATE-ZIP.PS1, line 135 at r3](https://reviewable.io:443/reviews/pmem/nvml/1948#-KpPg8hznFynnNG1Un5d:-KpUeUSmmb5OsGlJBSIT:b-896fix) ([raw file](https://github.com/pmem/nvml/blob/36d40a256277e31726a127c32dbb393199f89e8a/utils/CREATE-ZIP.PS1#L135)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary><blockquote>

did you mean $zipexp ?
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/nvml/1948)*
<!-- Sent from Reviewable.io -->

krzycz commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 1 of 78 files at r2, 1 of 2 files at r4, 49 of 50 files at r5.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/nvml/1948#-:-KptEaf3UBS868YthOjK:bnfp4nl)*
<!-- Sent from Reviewable.io -->
